opentelemetry/src/trace/span_context.rs

```diff
@@ -457,15 +468,18 @@ pub struct SpanContext {
 }
 
 impl SpanContext {
+    /// An invalid span context
+    pub const NONE: SpanContext = SpanContext {
+        trace_id: TraceId::INVALID,
+        span_id: SpanId::INVALID,
+        trace_flags: TraceFlags::NOT_SAMPLED,
+        is_remote: false,
+        trace_state: TraceState::NONE,
+    };
+
     /// Create an invalid empty span context
     pub fn empty_context() -> Self {
-        SpanContext::new(
```

Comment:
There are a lot of places using `empty_context()` that can be replaced with `SpanContext::NONE`.  I haven't changed them in this PR, but my plan would be to change them all in a future PR and then remove the `empty_context()` function.  Anyone see any problems with that approach?